apps/server/src/modules/workspaces/types.ts

```diff
@@ -0,0 +1,6 @@
+export enum Permission {
+  Read = 0,
+  Write = 1,
+  Admin = 10,
```

Comment:
```suggestion
  Admin = 100,
  Owner = 1000,
```

Reply:
The ones migrated from https://github.com/toeverything/OctoBase/blob/74867b7cbb96c3f67d362ac3db84c160d23fa711/libs/cloud-database/src/model.rs#L226